pages/demos/generate_and_detect_demo.py

```
@@ -1,6 +1,5 @@
 import streamlit as st
-from utils.extractor import extract_article_text
-from utils.language import detect_language
+from pages.components.generate_and_detect_component import generate_and_detect
```

Pylint (3.3.7)

[error] Unable to import 'pages.components.generate_and_detect_component'

(E0401)